src/components/endpoints/ImportProfile.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import useWeb3 from '@/compositions/useWeb3'
+import { ref } from 'vue'
+import Notifications from '@/components/Notification.vue'
+import useNotifications from '@/compositions/useNotifications'
+
+const { notification, clearNotification, hasNotification, setNotification } =
+  useNotifications()
+
+const { isAddress } = useWeb3()
+const controllerAddress = ref('')
```

Comment:
We should be importing the UP address not the controller address

Reply:
Thanks for the correction @olusoladavid